Makefile

```diff
@@ -14,7 +16,10 @@ install:
 install-dependencies:
 	go get -u github.com/stretchr/testify/...
 install-tools:
+
 test:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
+test-full:
+	$(ROOT_DIR)/scripts/test-full.sh
 test-verbose:
```

Comment:
"test-verbose" is just "test" but with verbose output. "test-full" adds should add additional tests, like the memory sanitizer, and later maybe integration/system tests. So I think it would be wiser to just call test-verbose in test-full, and then later activate the memory sanitzer check when it is ready. I just wanted to have a single make target for the CI and I do not want to include additional checks into test/test-verbose. Now that you mentioned it, I am also voting to remove the -race argument from test/test-verbose.